crates/edr_provider/src/error.rs

```diff
@@ -305,7 +305,7 @@ impl std::fmt::Display for TransactionFailureWithTraces {
     }
 }
 
-/// Wrapper around [`revm_primitives::Halt`] to convert error messages to match
+/// Wrapper around [`ExecutionResult::Halt`] to convert error messages to match
```

Comment:
```suggestion
/// Wrapper around [`revm_primitives::HaltReason`] to convert error messages to match
```

Reply:
Did this work for you locally? I think it didn't work for me because `revm_primitives` is not a dependency in `edr_provider`.

Reply:
I didn't test it locally. We re-export `revm_primitives::HaltReason` in `edr_evm`, so the local use statement is `edr_evm::HaltReason`. I didn't realise that doctest validates with local includes.

Reply:
```[`edr_evm::HaltReason`]``` does work.

Reply:
With that change, feel free to merge :)